controllers/apply/get-advice/form.js

```diff
@@ -0,0 +1,398 @@
+            schema: Joi.when('projectCountry', {
+                is: Joi.array().min(2),
+                then: Joi.any().strip()
+            }).when('projectCountry', {
+                is: Joi.array()
+                    .items(
+                        Joi.string()
+                            .only('scotland')
+                            .required()
+                    )
+                    .required(),
+                then: Joi.number()
+                    .integer()
+                    .required()
+                    .min(3)
+                    .max(5),
+                otherwise: Joi.number()
+                    .integer()
+                    .required()
+                    .min(1)
+                    .max(5)
+            }),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: 'Select a project duration',
+                        cy: ''
+                    })
+                }
+            ]
+        },
+        projectIdea: new TextField({
+            name: 'projectIdea',
+            label: localise({
+                en: 'What would you like to do?',
+                cy: ''
+            }),
+            type: 'textarea',
+            schema: Joi.string()
+                .minWords(50)
+                .maxWords(500)
+                .required(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: `Tell us about your idea`,
+                        cy: ``
+                    })
+                },
+                {
+                    type: 'string.minWords',
+                    message: localise({
+                        en: `Answer must be at least 50 words`,
+                        cy: `Rhaid i’r ateb fod yn o leiaf 50 gair`
+                    })
+                },
+                {
+                    type: 'string.maxWords',
+                    message: localise({
+                        en: `Answer must be no more than 500 words`,
+                        cy: `Rhaid i’r ateb fod yn llai na 500 gair`
+                    })
+                }
+            ]
+        }),
+        organisationLegalName: new TextField({
+            name: 'organisationLegalName',
+            label: localise({
+                en: 'What is the full legal name of your organisation?',
+                cy: ''
+            }),
+            explanation: localise({
+                en: oneLine`This must be as shown on your governing document.
+                    Your governing document could be called one of several things,
+                    depending on the type of organisation you're applying
+                    on behalf of. It may be called a constitution, trust deed,
+                    memorandum and articles of association,
+                    or something else entirely.`,
+                cy: ``
```

Comment:
We have this in the translations too – I think it's worth getting them all in now where they exist, unless we know these labels will change (in which case sorry, ignore me!)